react/components/EXPERIMENTAL_Table/DataTable/Row.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { FC, useState } from 'react'
+
+import Cell from './Cell'
+import useTableContext from '../hooks/useTableContext'
+import { NAMESPACES } from '../constants'
+import RowContainer from './RowContainer'
+
+const Row: FC<RowProps> = ({ rowData, index }) => {
+  const { visibleColumns } = useTableContext()
+
+  const rowKey = `row-${index}`
+
+  const renderCells = () => {
```

Comment:
Why is this a function instead of being the return?

Reply:
Changed